script/init.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+command -v yarn >/dev/null || (echo "Yarn package manager not found in \$PATH, aborting." && exit 1)
+
+echo -e "Welcome to the Maji installer. This script will help you setup your new Maji project.\n"
+
+while [[ -z "$APP_PACKAGE" ]]
+do
+  read -p "Enter a package name: " APP_PACKAGE
+done
+
+while [[ -z "$APP_PATH" ]]
+do
+  read -p "Enter a directory: " APP_PATH
```

Comment:
would be nice to "absolutize" the path or make it clear that needs to be absolute. Not sure about portable ways to expand a path to something that does not exist yet... so maybe just creating and capturing the APP_PATH before before moving to TMP_DIR?

Reply:
Yeah that would be nice, but I think that's something the CLI would need to take care of, not this wrapper script. WDYT?

Reply:
I guess both then, otherwise in this case the cli would expand the path relative to the tmp dir. Regardless, I realize my remark goes beyond what this PR is addressing so feel free to :shipit:

Reply:
Yes. I will address this, but separately.